charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -303,7 +303,21 @@ tests:
       repository: chainguard/slim-toolkit-debug
       pullPolicy: IfNotPresent
       version: ""
-      tag: latest@sha256:99cafee4f14fe07a3298fcb7b90d4f0c396cba150b65d937856788b42ad83f79
+      tag: latest@sha256:b7b4cb6a241990f1caba892e061b6b4fa014d778e9608a94b97580a40ab361bb
+
+  step:
+    ## @param tests.step.image.registry The OCI registry to pull the image from
+    ## @param tests.step.image.repository The repository within the registry
+    ## @param tests.step.image.pullPolicy The image pull policy
+    ## @param tests.step.image.version This value is deprecated in favor of tag. (Will be removed in a future release)
+    ## @param tests.step.image.tag Overrides the image tag whose default is the chart appVersion
+    ##
+    image:
+      registry: ""
+      repository: smallstep/step-cli
+      pullPolicy: IfNotPresent
+      version: ""
+      tag: latest@sha256:071f01f10ccd3aedeb65362b50a873dacd41b59b4d7e60ad46485e7eceec834e
```

Comment:
tag should be: 0.25.1

and, can you please add the following to the "spiffe-oidc-discovery-provider/values.yaml" section under file .github/tests/images.json
```
    {
      "query": "tests.step.image",
      "filter": "^[0-9]\\+\\.[0-9]\\+\\.[0-9]\\+$",
      "sort-flags": ["-t", ".", "-k1,1n", "-k2,2n", "-k3,3n"]
    },
```

This will ensure it receives automatic updates.